source/benchmarking/benchmarking.md

```diff
@@ -410,36 +416,144 @@ Dataset size: For score purposes, the dataset size for a task is the size of the
+| Before task | Drop the `corpus` collection. Create an empty `corpus` collection with the `create` command. Construct a Collection object for the `corpus` collection to use for insertion.                                                                                                                                       |
+| Do task     | Do an ordered `Client::bulkWrite` in the same namespace ("perftest.corpus") with the list of write models.                                                                                                                                                                                                                   |
+| After task  | n/a                                                                                                                                                                                                                                                                                                                |
+| Teardown    | Drop the `perftest` database.                                                                                                                                                                                                                                                                                      |
+
+#### Small doc Collection BulkWrite Mixed Operations
+
+Summary: This benchmark tests driver performance of a `Collection::bulkWrite` operation with small documents and mixed operations (e.g. insert, replace and delete).
+
+Dataset: The dataset, designated SMALL_DOC consists of a JSON document with an encoded length of approximately 250
+bytes.
+
+Dataset size: For score purposes, the dataset size for a task is the size of the single-document source file (275 bytes)
+times 20,000 operations (insert + replace), which equals 5,500,000 bytes or 5.5 MB.
+
+| Phase       | Description                                                                                                                                                                                                                                                                                                       |
+| ----------- |-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Setup       | Construct a MongoClient object. Drop the `perftest` database. Load the SMALL_DOC dataset into memory as a language-appropriate document type (or JSON string for C). Make 10,000 copies of the document. DO NOT manually add an `_id` field; leave it to the driver or database. Construct a list of write models with insert, replace and delete operations for each copy of the document. |
```

Comment:
I think we need more details here: ordering and explicit count for each operation kind.

Reply:
Yep will do! In the C# benchmarks, my proposed sequence involves performing an insert operation, followed by a replace, and then a delete operation for each document copy. Both the replace and delete operations use empty filters as well.